effekt/shared/src/main/scala/effekt/symbols/builtins.scala

```diff
@@ -82,9 +82,7 @@ object builtins {
   lazy val globalRegion = ExternResource(name("global"), TRegion)
 
   val rootTerms: Map[String, TermSymbol] = Map(
```

Comment:
I'm keeping the rest of this file (so we're still creating `TState.get` and `TState.put`) in case we want to do revive this part of the codebase later.